build-data/paper.at

```diff
@@ -360,3 +360,8 @@ public net.minecraft.world.entity.projectile.Projectile leftOwner
 # Teleport API
 public net.minecraft.server.network.ServerGamePacketListenerImpl internalTeleport(DDDFFLjava/util/Set;Z)V
 
+# Player Warden Warning API
```

Comment:
Access transformers are kind of being phased out for uses like this. In general, just modify the fields in the patch .

Reply:
Thought the trend was to get everything to the transformers... :-D nvm, resolved and rebased